src/fill/alloy.rs

```diff
@@ -456,6 +456,15 @@ mod tests {
+        let tx = TxEnvelope::decode(&mut raw_tx.as_slice()).unwrap();
+
+        let _ = test_trevm().fill_cfg(&NoopCfg).fill_block(&NoopBlock).fill_tx(&tx);
```

Comment:
can we add some assertions? ideally the other tests will in the future also make assertions

Reply:
might as well do it now. Added a few more assertion to all tests. Could be a bit more thorough, but this should be sufficient for basic checks